jython/AddPowerButton.py

```diff
@@ -0,0 +1,52 @@
+# Sample script to add a button to the main
+# JMRI application window that controls layout power
+#
+# Author: Bob Jacobsen, copyright 2007, 2024
+# Part of the JMRI distribution
+
+#
+# NOTE: This script does not support DecoderPro. Use the recommended
+# method if you need to add a script to a button in DecoderPro.
```

Comment:
What is the recommended method to add a button in DecoderPro? If it's documented on the web, a link here would be nice.